package-lock.json

```diff
@@ -1285,7 +1285,7 @@
     },
     "buffer-crc32": {
       "version": "0.2.13",
-      "resolved": "https://registry.npmjs.org/buffer-crc32/-/buffer-crc32-0.2.13.tgz",
+      "resolved": "https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/buffer-crc32/-/buffer-crc32-0.2.13.tgz",
```

Comment:
Don't forget to remove `repository.engineering.redhat.com/nexus/repository/` segment from the resolved entry.

Reply:
fixed